azure-pipelines.yml

```diff
@@ -0,0 +1,77 @@
+name: k8sUIWebApp_$(SourceBranchName)_$(Date:MMMd)$(Rev:.r)
+
+# PR trigger
+pr:
+- master
+
+# CI trigger
+trigger:
+- master
+
+jobs:
+- job: Build
+  timeoutInMinutes: 15
+  cancelTimeoutInMinutes: 2
+  pool:
+    name: 'Hosted VS2017'
+
+  steps:
+    - task: NodeTool@0
+      displayName: 'Use Node version'
+      inputs:
+        versionSpec: '8.4'
+
+    - task: Npm@1
+      displayName: 'npm install'
+      inputs:
+        command: install
+
+    - task: Npm@1
+      displayName: 'Build source'
+      inputs:
+        command: custom
+        customCommand: 'run build'
+
+- job: 'Publish'
```

Comment:
can we not pass the build outpath path (dist folder) to this job from the previous one so that we needn't build it all over again ?

Reply:
now we have only one, so that we can skip publish for PR